src/tools/roc_conv/main.cpp

```diff
@@ -68,23 +69,32 @@ int main(int argc, char** argv) {
 
     pipeline::ConverterConfig config;
 
-    if (args.frame_size_given) {
-        if (args.frame_size_arg <= 0) {
-            roc_log(LogError, "invalid --frame-size: should be > 0");
+    if (args.frame_length_given) {
+        if (!core::parse_duration(args.frame_length_arg, config.internal_frame_length)) {
+            roc_log(LogError, "invalid --frame-length: frame length unspecified");
+            return 1;
+        }
```

Comment:
`frame length unspecified` is not correct. if parse_duration() returned error, it means that frame length *was* specified, but in an invalid format. So it should be something like: `invalid --frame-length: bad format`. Same for other tools.

Reply:
done.